app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self.favourite_dish = favourite_dish
+
+    def eat_favourite_dish(self) -> str:

```

Comment:
Invalid type hint for return type. It returns None
```suggestion
    def eat_favourite_dish(self) -> None:

```